source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -221,7 +228,7 @@ The following example performs these actions:
 - Finds all documents with "Pizza" in the ``cuisine`` field
```

Comment:
S: cant comment there, but change wording to:
This example performs the following actions: